card.io/src/main/jni/nativeRecognizer.cpp

```diff
@@ -287,8 +296,22 @@ void setDetectedCardImage(JNIEnv* env, jobject jCardResultBitmap, IplImage* card
 
     IplImage* cardResult = cvCreateImageHeader(cvSize(bmInfo.width, bmInfo.height), IPL_DEPTH_8U, 4);
     cvSetData(cardResult, pixels, bmInfo.stride);
-
     dmz_YCbCr_to_RGB(cardY, bigCb, bigCr, &cardResult);
+
+    // blur cardnumber
```

Comment:
I will check this on monday. If i get functionality down to DMZ and get it running on Android, can you do the iOS adaptation/testing for me? Should not be more than adding the parameter.